addon-test-support/should-force-audit.ts

```diff
@@ -12,6 +12,10 @@ export function setEnableA11yAudit(enabled: boolean = false) {
   window.history.replaceState(null, '', url.href);
 }
 
+/**
+ * Evaluates whether or not to run the audit. This functionality is enabled by
+ * the presence of an `enableA11yAudit` query parameter passed to the test suite.
```

Comment:
This prose doesn't seem quite right. This doesn't evaluate if we should or should not run the audit, it evaluates if we should run the audit **regardless** of the result of the provided `shouldAudit` invocation strategy.

Reply:
Updated.